azure-pipelines.yml

```diff
@@ -12,12 +12,14 @@ resources:
       type: github
       name: xamarin/XamarinComponents
       endpoint: xamarin
+      ref: refs/heads/dev/remove-manifest
 
 jobs:
   - template: .ci/build.yml@components
     parameters:
       timeoutInMinutes: 360
-      buildType: 'manifest'
+      buildType: 'directories'
+      targetsFilter: 'ci'
```

Comment:
The new target that will be built will be `ci`, so most probably any older builds will fail. But, this is not really an issue as they will only be building if they were changed - and if they are changed, then they need to be migrated.